packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -183,7 +183,6 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		...props,
 		ref: mergedRefs,
 		id: `block-${ clientId }${ htmlSuffix }`,
-		role: 'document',
```

Comment:
Could try to change this back to group role but would need to land this first.

https://github.com/WordPress/gutenberg/pull/47276

If not, JAWS compatibility may break.